gettingStarted/sysElevenStackKickstart.yaml

```diff
@@ -64,7 +64,8 @@ resources:
               mkdir /home/syseleven/.ssh
               curl -s http://169.254.169.254/latest/meta-data/public-keys/0/openssh-key > /home/syseleven/.ssh/authorized_keys
               cat <<EOF> /home/syseleven/.ssh/config
-              Host 185.56.* 
+              # Accept all S11 FloatingIPs HostKeys
```

Comment:
Why are you doing this? This is insecure and should not be done. This is basically disabling SSHs man in the middle protection completely.

Also this is an unrelated change. Please do not put many unrelated changes in one commit, and defenitely not in one merge request!